pkgs/by-name/hy/hyprland-qtutils/package.nix

```
@@ -0,0 +1,59 @@
+{
+  lib,
+  stdenv,
+  cmake,
+  pkg-config,
+  hyprutils,
+  pciutils,
+  qt6,
+  fetchFromGitHub,
+  aquamarine,
+  hyprlang,
+  hyprcursor,
+  hyprgraphics,
+}:
+let
+  version = "0.1.1";
+  inherit (lib.sources) cleanSource cleanSourceWith;
+  inherit (lib.strings) hasSuffix makeBinPath;
+in
+stdenv.mkDerivation {
```

```suggestion
stdenv.mkDerivation (finalAttrs: {
```

With this we can reference `finalAttrs.version` inside `fetchFromGitHub`.